pywr/solvers/cython_glpk.pyx

```diff
@@ -395,48 +372,12 @@ cdef class CythonGLPKSolver:
             'number_of_nodes': len(self.all_nodes)
         }
 
-    cdef _init_basis_arrays(self, model):
-        """ Initialise the arrays used for storing the LP basis by scenario """
-        cdef int nscenarios = len(model.scenarios.combinations)
-        cdef int nrows = glp_get_num_rows(self.prob)
-        cdef int ncols = glp_get_num_cols(self.prob)
-
-        self.row_stat = np.empty((nscenarios, nrows), dtype=np.int32)
-        self.col_stat = np.empty((nscenarios, ncols), dtype=np.int32)
-
-    cdef _save_basis(self, int global_id):
-        """ Save the current basis for scenario associated with global_id """
-        cdef int i
-        cdef int nrows = glp_get_num_rows(self.prob)
-        cdef int ncols = glp_get_num_cols(self.prob)
-
-        for i in range(nrows):
-            self.row_stat[global_id, i] = glp_get_row_stat(self.prob, i+1)
-        for i in range(ncols):
-            self.col_stat[global_id, i] = glp_get_col_stat(self.prob, i+1)
-
-    cdef _set_basis(self, int global_id):
-        """ Set the current basis for scenario associated with global_id """
-        cdef int i, nrows, ncols
-
-        if self.is_first_solve:
-            # First time solving we use the default advanced basis
-            glp_adv_basis(self.prob, 0)
-        else:
-            # otherwise we restore basis from previous solve of this scenario
-            nrows = glp_get_num_rows(self.prob)
-            ncols = glp_get_num_cols(self.prob)
-
-            for i in range(nrows):
-                glp_set_row_stat(self.prob, i+1, self.row_stat[global_id, i])
-            for i in range(ncols):
-                glp_set_col_stat(self.prob, i+1, self.col_stat[global_id, i])
-
     def reset(self):
```

Comment:
Can this be done in the base class? Looks the same between the two implementations?